frontend/src/pages/pipelines/global/modelCustomization/startRunModal/StartRunModal.tsx

```diff
@@ -40,35 +52,32 @@ const StartRunModal: React.FC<StartRunModalProps> = ({
           isSubmitDisabled={!canContinue || !loaded}
         />
       }
-      variant="medium"
+      variant={ModalVariant.medium}
       data-testid="start-run-modal"
+      hasNoBodyWrapper
     >
-      <Form>
-        <Stack hasGutter>
-          {loadError && (
-            <StackItem>
-              <Alert variant="danger" title="Error loading model data">
-                {loadError.message}
-              </Alert>
-            </StackItem>
-          )}
-          <StackItem>
-            Fine-tune your models to improve their performance, accuracy, and task specialization,
-            using the{' '}
-            <Button
-              data-testid="lab-method"
-              variant="link"
-              isInline
-              component="a"
-              style={{ textDecoration: 'none' }}
-              onClick={() => {
-                // TODO: Link to documentation
-              }}
-            >
-              LAB method
-            </Button>
-            . Before creating a run, a taxonomy is needed and a teacher and judge model must be
-            configured.{' '}
+      <ModalHeader
+        title="Start a LAB-tuning run"
+        description={
+          <>
+            <p>
```

Comment:
Small suggestion: consider using`<Content component={ContentVariants.p}>` instead of `<p>`